webapp/src/Service/ImportExportService.php

```diff
@@ -813,6 +821,73 @@ public function importTeamsJson(array $data, ?string &$message = null, ?array &$
         return $this->importTeamData($teamData, $saved);
     }
 
+    /**
+     * Import accounts JSON
+     *
+     * @param User[]|null $saved The saved users
+     */
+    public function importAccountsJson(array $data, ?string &$message = null, ?array &$saved = null): int
+    {
+        $teamRole     = $this->em->getRepository(Role::class)->findOneBy(['dj_role' => 'team']);
+        $juryRole     = $this->em->getRepository(Role::class)->findOneBy(['dj_role' => 'jury']);
+        $adminRole    = $this->em->getRepository(Role::class)->findOneBy(['dj_role' => 'admin']);
+        $juryCategory = $this->em->getRepository(TeamCategory::class)->findOneBy(['name' => 'Jury']);
+        if (!$juryCategory) {
+            $juryCategory = new TeamCategory();
+            $juryCategory
+                ->setName('Jury')
+                ->setSortorder(100)
+                ->setVisible(false)
+                ->setExternalid('jury');
+            $this->em->persist($juryCategory);
+            $this->em->flush();
+        }
+        $accountData = [];
+        foreach ($data as $idx => $account) {
+            $juryTeam = null;
+            $roles    = [];
+            switch ($account['type']) {
+                case 'admin':
+                    $roles[] = $adminRole;
+                    break;
+                case 'judge':
+                    $roles[]  = $juryRole;
+                    $roles[]  = $teamRole;
+                    $juryTeam = [
+                        'name'              => $account['name'] ?? $account['username'],
+                        'externalid'        => $account['externalid'] ?? $account['username'],
+                        'category'          => $juryCategory,
+                        'publicdescription' => $account['name'] ?? $account['username'],
+                    ];
+                    break;
+                case 'team':
+                    $roles[] = $teamRole;
+                    break;
+                case 'analyst':
+                case 'staff':
+                    // Ignore type analyst and staff for now. We don't have a useful mapping yet.
+                    continue 2;
```

Comment:
Should there be a warning that those were not imported?

Reply:
I don’t think so. We don’t do it for the tsv either. We also don’t really have a way to report it currently